src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.cpp

```diff
@@ -166,6 +166,82 @@ tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
   return deriv_spatial_metric;
 }
 
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::ii<DataType, SpatialDim, Frame> time_deriv_of_spatial_metric(
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& d_spacetime_metric,
+    const tnsr::iaa<DataType, SpatialDim, Frame>& phi,
+    const tnsr::aa<DataType, SpatialDim, Frame>& pi,
+    const Scalar<DataType>& gamma1) noexcept {
+  const auto gamma1p1 = 1.0 + gamma1.get();
+  auto three_index_constraint =
+    make_with_value<tnsr::iaa<DataType, SpatialDim, Frame>>(get(lapse), 0.);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    for (size_t a = 0; a < SpatialDim+1; ++a) {
+      for (size_t b = a; b < SpatialDim+1; ++b) {
+        three_index_constraint.get(i, a, b) =
+            d_spacetime_metric.get(i, a, b) - phi.get(i, a, b);
+      }
+    }
+  }
+  auto shift_dot_three_index_constraint =
+    make_with_value<tnsr::aa<DataType, SpatialDim, Frame>>(get(lapse), 0.);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    for (size_t a = 0; a < SpatialDim+1; ++a) {
+      for (size_t b = a; b < SpatialDim+1; ++b) {
+        shift_dot_three_index_constraint.get(a, b) +=
+            shift.get(i) * three_index_constraint.get(i, a, b);
+      }
+    }
+  }
+  // Equation for dt_spatial_metric
+  auto dt_spatial_metric =
+    make_with_value<tnsr::ii<DataType, SpatialDim, Frame>>(get(lapse), 0.);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    for (size_t j = i; j < SpatialDim; ++j) {
+      dt_spatial_metric.get(i, j) = -get(lapse) * pi.get(i+1, j+1)
+        + gamma1p1 * shift_dot_three_index_constraint.get(i+1, j+1);
```

Comment:
Where is this dt_spatial_metric being used?  If this is used to compute time derivatives of the gauge source function, shouldn't this be computed without gamma1 (i.e. hardcoding gamma1 = -1)?  Because I thought that "derivatives of H" are understood to include pi and phi instead of explicit derivatives of the metric.

Reply:
Initially I had thought of writing this function including constraint damping terms, and use it with `gamma1 = -1, dt_spatial_metric = 0` when calling it within damped harmonic gauge-related computations. That way, this function would have been useful if one needed the RHS for `g` somewhere. I see now, however, that doing so was confusing. I've fixed this by removing constraint damping terms.